Modules/Core/Services/ImportService.php

```
@@ -0,0 +1,139 @@
+
+    /**
+     * Import data from CSV file.
+     *
+     * @param string $file CSV filename
+     * @param string $table Target table name
+     *
+     * @return array Array of imported IDs
+     *
+     * @legacy-function importData
+     */
+    public function importData(string $file, string $table): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$file'. (undefined)

(UnusedFormalParameter)